entrypoint.sh

```diff
@@ -43,28 +79,26 @@ cdktf synth --output "$ER_OUTDIR"
 $TERRAFORM_CMD init
 
 if [[ $ACTION == "Apply" ]]; then
-    if [[ $DRY_RUN == "True" ]]; then
-        $TERRAFORM_CMD plan -out=plan --lock=false
-        if [ -f "validate_plan.py" ]; then
-            $TERRAFORM_CMD show -json "$CDKTF_OUT_DIR"/plan > "$CDKTF_OUT_DIR"/plan.json
-            python3 validate_plan.py "$CDKTF_OUT_DIR"/plan.json
-        fi
-    elif [[ $DRY_RUN == "False" ]]; then
+    $TERRAFORM_CMD plan -out=plan --lock=false
```

Comment:
I would prefer not to run `lock=false` along with `DRY_RUN=False`.  
if the module is potentially going to apply changes, I think it's better to lock the state.

Reply:
👍 changed in [029293c](https://github.com/app-sre/er-base-cdktf/pull/12/commits/029293c5ccde123deb5a72e14e12dff1091876d5)